lib/themes/dosomething/paraneue_dosomething/js/campaign/prototype.js

```diff
@@ -58,23 +57,18 @@ $(document).on('click', '.js-kudos-button', function(event) {
   const dataKid = $el.attr('data-kid');
   const method = dataKid ? 'DELETE' : 'POST';
 
-  // if (setting('dsKudosReactions.sendToRogue')) {
-  //   makeKudosRequestToRogue(setting('dsKudosReactions.northstarId'), setting('dsKudosReactions.postId'))
-  // } else {
   makeKudosRequest($el, dataKid)
     .done((response) => {
       const kid = get(response, '0.kid', '');
 
-      if (method === 'POST' && !kid) {
+      if (method === 'POST' && (!kid || kid === 'unliked')) {
```

Comment:
@DFurnes actually found a hacky way to add this conditional back in! Instead of just doing a `return`, set the `kid` to `unliked` and check that here. Does this work?